test/test_prototype_transforms_consistency.py

```diff
@@ -840,3 +840,681 @@ def test_aa(self, inpt, interpolation):
         output = t(inpt)
 
         assert_equal(expected_output, output)
+
+
+class TestRefDetTransforms:
+    def make_datapoints(self, with_mask=True):
+        size = (600, 800)
+
+        pil_image = PIL.Image.new("RGB", size[::-1], 123)
+        target = {
+            "boxes": make_bounding_box(image_size=size, format="XYXY", extra_dims=(22,), dtype=torch.float),
+            "labels": features.Label(torch.randint(0, 81, size=(22,))),
+        }
+        if with_mask:
+            target["masks"] = features.SegmentationMask(torch.randint(0, 2, size=(22, *size), dtype=torch.long))
```

Comment:
See #6579

```suggestion
            target["masks"] = features.Mask(torch.randint(0, 2, size=(22, *size), dtype=torch.long))
```

Reply:
Better yet, use the common utilities:

```suggestion
            target["masks"] = make_detection_mask(size=size, extra_dims=[(22,)], dtype=torch.long)
```

This also applies to the other comments like the one above.